### PR TITLE
Icon folder cleaning wave one

### PR DESCRIPTION
**Pull request by YesterdaysPromise**
## About The Pull Request

Due to a mental breakdown caused by unfathomable abomination that is icons folder, I swore to myself to one day clean it. Today is kind of that day. Been at it for around 6, you gotta understand I need a rest. I tracked most changes  in descriptions of commits if you are looking for details.

## Why It's Good For The Game

Saner spriters make better sprites. And also, just helps keep track of things.

## Changelog

:cl:
image: added sprites for different variants of scrolls.
image: modified couple posters with ghost pixels.
/:cl:

### CODE REVIEW

**Comment by ZephyrTFA** [comment]
Changelog is for player visible changes only

**Comment by YesterdaysPromise** [comment]
> 

fair, fixed.

File: tgstation.dme [hunk 1]
```diff
@@ -5384,8 +5384,8 @@
 #include "code\ze_genesis_call\genesis_call.dm"
 #include "interface\interface.dm"
 #include "interface\menu.dm"
-#include "interface\skin.dmf"
 #include "interface\stylesheet.dm"
+#include "interface\skin.dmf"
```
Comment (OrionTheFox) [inline tgstation.dme:5388]:
```suggestion
#include "interface\skin.dmf"
#include "interface\stylesheet.dm"
```
This keeps trying to change itself and the code doesnt like when its changed so we have to make sure it doesnt change itself 💀

**Comment by ShizCalev** [comment]
*laughs in icon unit testing*

**Comment by YesterdaysPromise** [comment]
> _laughs maniacally in icon unit testing_

huh?

**Comment by ChungusGamer666** [comment]
heroic

**Comment by YesterdaysPromise** [comment]
alright seeing just how slow it checks, this will be the end of wave one

**Pull request merged**